src/BrightScriptDebugSession.ts

```diff
@@ -15,13 +15,33 @@ import {
     Thread,
     Variable
 } from 'vscode-debugadapter';
+
 import { DebugProtocol } from 'vscode-debugprotocol';
 
-//becomes this
-import {
-    EvaluateContainer,
-    RokuAdapter
-} from './RokuAdapter';
+import { EvaluateContainer, RokuAdapter } from './RokuAdapter';
```

Comment:
Per the shiny new [developer guidelines](https://github.com/TwitchBronBron/vscode-brightscript-language/blob/master/developer-guidelines.md), each import item should have its own line. So this line should actually be this: 

```
import {
    EvaluateContainer,
    RokuAdapter
} from './RokuAdapter';
```